misc/utils-db/src/errors.rs

```diff
@@ -23,14 +23,14 @@ pub enum DbError {
 
 pub type Result<T> = std::result::Result<T, DbError>;
 
-#[cfg(feature = "wasm")]
+#[cfg(feature = "js")]
 impl From<DbError> for wasm_bindgen::JsValue {
     fn from(value: DbError) -> Self {
         value.to_string().into()
     }
 }
 
-#[cfg(feature = "sqlite")]
+#[cfg(not(feature = "js"))]
```

Comment:
This should be `all(feature = "sqlite", not(feature="js")`, because if `sqlite` feature is not specified, the `sqlx` dependency will be missing.

Reply:
That is correct... but I am hopeful that this package will be dissolved and the `chain_db` will be moved to common/db with partitioned interfaces, or just the model layer.